specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/compute.json

```diff
@@ -4502,7 +4502,7 @@
       ],
       "x-ms-enum": {
         "name": "DiskCreateOptionTypes",
-        "modelAsString": false
+        "modelAsString": true
```

Comment:
@olydis Does this constitute a breaking change for C#?

Reply:
yes, this change is needed in order to add new create types, in addition I'm changing it in disk.json so the two need to match

Reply:
@jhendrixMSFT yes (and for other languages)